packages/x-date-pickers/src/PickersDay/PickersDay.tsx

```diff
@@ -135,7 +135,7 @@ const styleArg = ({ theme, ownerState }: { theme: Theme; ownerState: OwnerState
   // explicitly setting to `transparent` to avoid potentially getting impacted by change from the overridden component
   backgroundColor: 'transparent',
   color: (theme.vars || theme).palette.text.primary,
-  '@media (pointer: fine)': {
+  '@media (hover: hover) and (pointer: fine)': {
```

Comment:
Is there a reason to not use `DEFAULT_DESKTOP_MODE_MEDIA_QUERY` here (hand in `Clock`) ?

Reply:
No reason, just didn't replace, because it was not used initially and is somewhat of a different context. 🤔